src/main/java/org/komamitsu/fluency/buffer/Buffer.java

```diff
@@ -86,7 +86,7 @@ protected void saveBuffer(List<String> params, ByteBuffer buffer)
         if (fileBackup == null) {
             return;
         }
-        LOG.info("Saving buffer: params={}, buffer={}", params, buffer);
+        LOG.debug("Saving buffer: params={}, buffer={}", params, buffer);
```

Comment:
This log message is a bit important and appears only when file backup mode is enables, I want to leave it `info`.